BridgeSDK/BridgeAPI/BridgeAPITypes/SBBSchedule.m

```diff
@@ -33,6 +33,19 @@ @implementation SBBSchedule
 
 #pragma mark Abstract method overrides
 
-// Custom logic goes here.
+- (NSString *) description
```

Comment:
It makes sense to provide "description" methods for our SBBxxx classes and not make AppCore implement them. Rather than doing it in each class though, I'd suggest putting a description method on the parent SBBBridgeObject class, that (for example) returns [[self dictionaryRepresentation] description] and get all of them covered at once.